ClientApp/viking-vault/src/display_user_list/AddMoneyModal.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import './AddMoney.css';
+
+interface IAddMoneyModalProps{
+    open: boolean;
+    userName: string;
+    closeModal: any;
+    addMoney: any;
+}
+
+interface IAddMoneyModalState{
+    amount: string;
+}
+
+class AddMoneyModal extends React.Component<IAddMoneyModalProps, IAddMoneyModalState>{ 
+    state = {
+            amount : ""
+        };
+
+    private resetAmountField(){
+        this.setState({
+            amount : ""
+        });
+    }
+    private closeModal = () =>{
+        this.resetAmountField();
+        this.props.closeModal(false);
```

Comment:
Why does this accept a boolean as an argument? I would expect that when I call a function named `closeModal` to just call it without any arguments.